### ISSUE TITLE
Disable right click mouse events in the "Multi-Account Containers" panel

### ISSUE THREAD

**Issue by Softvision-MariusComan** [mozilla/multi-account-containers#887]
**[Affected versions]:**
- Multi-Account Containers 4.0.3
- Firefox 56 and up

**[Affected Platforms]:**
- All Windows
- All Mac
- All Linux

**[Prerequisites]:**
- Have a Firefox profile with the latest "Multi-Account Containers" add-on version (4.0.3) installed.

**[Steps to reproduce]:**
1. Open the browser with the profiles from prerequisites.
2. Click the "Multi-Account Containers" from the toolbar.
3. Right click in the "Multi-Account Containers" panel and observe the behavior.

**[Expected result]:**
- Nothing happens.

**[Actual result]:**
- A context menu is opened.

**[Notes]:**
- If a website is opened in a container tab, the context menu from the panel has the "Multi-Account Containers" option, which does not work at all, even if the UI suggests that it does.
- A few options from the context menu are related to the opened page, while others affect the "Multi-Account Containers" panel itself.
- The issue is also reproducible with the TxP version of "Multi-Account Containers".
- Attached a screen recording of the issue:
![context menu](https://user-images.githubusercontent.com/31243259/31118223-5b294d76-a835-11e7-8d57-7b2d85d88706.gif)

**Comment by jonathanKingston** [comment]
This appears to have been fixed in the latest Nightly or at least I can't reproduce? Could you confirm?

**Comment by Softvision-MariusComan** [comment]
The "Firefox Multi-Account Containers" option is no longer displayed on the "Multi-Account Containers" panel.
However the other options such as "select all", "save as", "View page source", are still displayed.
I've tested this issue on Windows 10 x64 with latest Nightly (58.0a1) build id 20171005220204 and the "Multi-Account Containers" 4.0.3 custom add-on installed.

**Comment by jonathanKingston** [comment]
Yeah this was my experience however the patch I have attached also doesn't fix anything. I think we should close this unless you see a more serious issue here?

In my actual Nightly browser which is at the latest, I see the issue. In my central build I don't. I'm going to leave this open for a few days to check. The save as appears to be native behaviour that we can't control.
